ccm_web/server/src/canvas/canvas.interfaces.ts

```diff
@@ -34,15 +34,20 @@ export interface TokenRefreshResponseBody extends TokenBaseResponseBody {}
 export interface CanvasCourseBase {
   id: number
   name: string
+  enrollment_term_id: number
 }
 
 export interface CanvasCourse extends CanvasCourseBase {
   course_code: string
 }
 
-export interface CanvasCourseSection {
+export interface CanvasCourseSectionBase {
   id: number
   name: string
+  course_id: number
+}
+
+export interface CanvasCourseSection extends CanvasCourseSectionBase {
   total_students: number
 }
```

Comment:
I think we need a `Base` because operations like `merge` and `unmerge` don't have access to `total_students`. Will double check that.

Reply:
Yeah... it is not returned as part of merge/unmerge sections. 

I think the course/section object in the FE is following the same object pattern as Backend. May be that should reflect this pattern? https://github.com/tl-its-umich-edu/canvas-course-manager-next/blob/main/ccm_web/client/src/models/canvas.ts#L6

Reply:
Yes, good point, I can change that. Hopefully it doesn't mess with @chrisrrowland too much.

Reply:
I added a bit more typing to the client `models/canvas.ts` (see https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/199/commits/a014d63d6431cd0621d27e802816c06fb7a44a3e) to reflect the server changes.

I'm going to leave the `api.ts` definitions to Chris, since he's already done some of that work and may have opinions on what parameters they accept, how they are named, but @chrisrrowland, please note the nuance that the merge and unmerge routes do not return `total_students`, so you may need to hold the original section objects in state and update the `course_id` (or do some other client magic).